scripts/generate_ssl_server_defaults.cr

```diff
@@ -56,6 +56,7 @@ File.open(DEFAULTS_FILE, "w") do |file|
       # available at #{guidelines.href}.
       #
       # See https://wiki.mozilla.org/Security/Server_Side_TLS for details.
+      @[Deprecated("Deprecated with no replacement. Prefer setting a security level, global system configuration, or build your own from #{url}")]
```

Comment:
*suggestion:* Link a human-readable version instead of the JSON format.

https://ssl-config.mozilla.org

Reply:
Remember to run this script to regenerate `src/openssl/ssl/defaults.cr` ;)

Reply:
:facepalm: